package-support-policy.md

```diff
@@ -0,0 +1,90 @@
+# Support Policy for .NET Packages
+
+NuGet packages owned by the .NET team are supported as long as they target at least one framework that is still actively supported.
```

Comment:
```suggestion
.NET library packages provide additional functionality that complements the .NET libraries and application frameworks like ASP.NET Core. The library packages have the same support model as the .NET versions that they target.

For example, [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging/#supportedframeworks-body-tab) supports multiple .NET versions via the frameworks it targets.
```

Reply:
I'm not sure how to write it, but it might be helpful to distinguish which version of the package is supported in that way.  Not all versions - only *a* version.

We'll continue to support a version of the package, if that package was removed from the product release and it still targets in support frameworks (eg: System.Memory).  We'll also support an additional version of a package if future versions of that package removed targeting for an in support framework (eg: Asp.NET Core on netfx).

Reply:
^ Still thinking of this feedback by Eric. I addressed all the other feedback.

Reply:
I see this doc has a "Servicing" and an "End of support" sections: https://github.com/dotnet/core/blob/main/release-policies.md#end-of-support

I'll try to integrate Eric's request based on that info.